distributions/openhab/src/main/resources/conf/services/runtime.cfg

```diff
@@ -284,3 +284,9 @@
 # org.openhab.webclient:maxThreadsShared = 80
 # org.openhab.webclient:minThreadsCustom = 20
 # org.openhab.webclient:maxThreadsCustom = 40
+
+######################### ITEM CHANNEL LINK REGISTRY ###########################
+
+# If enabled, the ItemChannelLink Registry will always try to apply any tags from linked Channels to the respective Items
+#
+#org.openhab.ItemChannelLinkRegistry:useTags=true
```

Comment:
AFAIK, generally the commented out setting should be the _default_ value as it applies without specifying the setting. So in this case, it should be written as `=false` because that's what happens without the setting.

See other settings, e.g. `org.openhab.voice:cacheSizeTTS`, `org.openhab.sitemap:groupMembersSorting`, autoApprove, autoIgnore, although this is not entirely consistent in this file, e.g. allowBasicAuth defaults to false. In this case, we should probably fix allowBasicAuth but in a different PR.